portal_spa/src/store/index.ts

```diff
@@ -93,7 +93,7 @@ const useStore = create<StateStore>((set) => ({
           accessToken: savedAccessToken,
           email: decoded.email,
           role: decoded.role,
-          organizationSlug: decoded.organizationSlug,
+          organizationSlugs: decoded.organizationSlugs || [],
```

Comment:
Would recommend nullish coalescing (using `??` instead of `||`) here. Otherwise the value of `organizationSlugs` could become a boolean instead of an array.